python_modules/dagster/dagster/_cli/__init__.py

```diff
@@ -29,6 +30,7 @@ def create_dagster_cli():
         "asset": asset_cli,
         "debug": debug_cli,
         "new-project": new_project_cli,
+        "project": project_cli
```

Comment:
we should remove "new-project" soon after adding this, right?

Reply:
yes! i also plan to talk to 1.0 folks about deprecation vs removal